vendor/engines/projects/app/controllers/projects/projects_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Projects
+
+  class ProjectsController < ApplicationController
+
+    admin_tab :all
+    before_filter { @active_tab = "admin_projects" }
+
+    load_and_authorize_resource
+
+    def index
+      @projects = current_facility.projects.paginate(page: params[:page])
+    end
+
+    def new
+      @project = Projects::Project.new(facility: current_facility)
+    end
+
+    def create
+      @project = current_facility.projects.new(project_params)
+      if @project.save
+        flash[:notice] =
+          I18n.t("controllers.projects.projects.create.success", project_name: @project.name)
```

Comment:
Does I18n sanitize the interpolation options? Probably not a big deal here, though; just something to watch out for.